src/components/tooltip/Tooltip.tsx

```diff
@@ -81,12 +83,15 @@ export class Tooltip extends React.Component<ITooltipProps, any> {
       {tooltipFooter}
     </BootstrapTooltip>;
 
-    return (
-      <OverlayTrigger {..._.omit(this.props, OVERLAY_PROPS_TO_OMIT) } overlay={tooltip}>
-        <span className={this.props.className}>
-          {this.props.children}
-        </span>
-      </OverlayTrigger>
-    );
+    if (this.props.title) {
+      return (
+        <OverlayTrigger {..._.omit(this.props, OVERLAY_PROPS_TO_OMIT) } overlay={tooltip}>
+          <span className={this.props.className}>
+            {this.props.children}
+          </span>
+        </OverlayTrigger>
+      );
+    }
+    return (<span>{this.props.children}</span>);
```

Comment:
Still need the span? If you do, should you add the same className as the span in the overlay?

Reply:
span removed